public/style/base/_typography.scss

```diff
@@ -56,3 +56,7 @@ a:visited {
   // box-shadow: 0px 1px 1px red;
   color: $brand-primary-darker;
 }
+
+a.blank:after {
+  content: '\01F5D7';
```

Comment:
no Mac OS support.Find a supported font family, or change the unicode character for better support?